src/main/java/org/jabref/logic/openoffice/oocsltext/CSLFormatUtils.java

```diff
@@ -0,0 +1,161 @@
+package org.jabref.logic.openoffice.oocsltext;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.citationkeypattern.BracketedPattern;
+import org.jabref.logic.citationstyle.CitationStyleOutputFormat;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.openoffice.ootext.OOText;
+import org.jabref.model.openoffice.ootext.OOTextIntoOO;
+
+import com.sun.star.text.XTextCursor;
+import com.sun.star.text.XTextDocument;
+import org.apache.commons.text.StringEscapeUtils;
+
+/**
+ * Contains utility constants and methods for processing of CSL citations as generated by methods of <a href="https://github.com/michel-kraemer/citeproc-javaciteproc-java">citeproc-java</a> ({@link org.jabref.logic.citationstyle.CitationStyleGenerator}).
+ * <p>These methods are used in {@link CSLCitationOOAdapter} which inserts CSL citation text into an OO document.</p>
+ */
+public class CSLFormatUtils {
+
+    // TODO: These are static final fields right now, should add the functionality to let user select these and store them in preferences.
+    public static final String DEFAULT_BIBLIOGRAPHY_TITLE = "References";
+    public static final String DEFAULT_BIBLIOGRAPHY_HEADER_PARAGRAPH_FORMAT = "Heading 2";
+    public static final CitationStyleOutputFormat OUTPUT_FORMAT = CitationStyleOutputFormat.HTML;
```

Comment:
I think, this should be separated by a newline from the constant before. Reason: This one should not be configurable, should it?

Reply:
Right, separated.